__tests__/business-text/business-text.mutation.test.js

```diff
@@ -0,0 +1,292 @@
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+
+const { BUSINESS_TEXT_NOT_FOUND } = require('../../error-messages/business-text.messages');
+const { newBusinessText, updatedBusinessText, notExistBusinessTextId } = require('./business-text.variables');
+require('dotenv')
+  .config();
+
+let businessText = null;
+let businessTextId = '';
+
+describe('Business text mutations test', () => {
+
+  test('#1 should add business text to database', async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation($businessText: BusinessTextInput!) {
+              addBusinessText(businessText: $businessText) {
+                  ... on BusinessText {
+                      _id
+                      code
+                      title {
+                          value
+                          lang
+                      }
+                      text {
+                          value
+                          lang
+                      }
+                  }
+                  ... on Error {
+                      message
+                      statusCode
+                  }
+              }
+          }
+      `,
+        variables: {
+          businessText: newBusinessText
+        },
+      })
+      .then(response => response)
+      .catch(e => e);
+
+    businessText = res.data.addBusinessText;
+    businessTextId = businessText._id;
+
+    expect(businessText)
+      .toHaveProperty('code', 'new-code');
```

Comment:
Please, use here values from the `newBusinessText` instead of plain hardcode literals. It would be much easier and safe to changes some values in `newBusinessText` and be sure that it is changes in tests too. Use hardcode values only for properties that you cannot take from `newBusinessText`